prettier/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Add git-clone
+kubectl -n ${tns} apply -f https://raw.githubusercontent.com/tektoncd/catalog/bf7e538778ce6c1ea8ebc72bd0d050362e63a716/git/git-clone.yaml
```

Comment:
very cool! Did not know the test framework allowed this!

Reply:
what is the issue doing it directly from the current repository ? like the other task are doing it ? i.e: 

https://github.com/tektoncd/catalog/blob/v1beta1/buildah/tests/pre-apply-task-hook.sh#L7-L8